.rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 120
```

Comment:
why?

Reply:
Commonly used value of max_width are 80, 100 or 120, choose for 120 makes my code clear. Samples:

```
// max_width = 120
let r = call(block_provider.clone(), state_provider.clone(), store.clone(), &reqchan)
```

```
// default
let r = call(
    block_provider.clone(), 
    state_provider.clone(), 
    store.clone(), 
    &reqchan
)
```